tests/test_pipelines/test_import_tiff.py

```diff
@@ -0,0 +1,64 @@
+"""
+Testing batch-to-eopatch pipeline
+"""
+import os
+
+import numpy as np
+import pytest
+from shapely.ops import unary_union
+
+from sentinelhub import BBox
+
+from eogrow.core.config import interpret_config_from_path
+from eogrow.pipelines.import_tiff import ImportTiffPipeline
+from eogrow.utils.testing import ContentTester, check_pipeline_logs, create_folder_dict, generate_tiff_file
+
+
+@pytest.fixture(scope="session", name="folders")
+def config_folder_fixture(config_folder, stats_folder):
+    return create_folder_dict(config_folder, stats_folder, "import_tiff")
+
+
+@pytest.mark.parametrize(
+    "experiment_name",
+    [
+        "import_tiff_temporal",
+        "import_tiff_timeless",
+        pytest.param("import_tiff_resized_new_size", marks=pytest.mark.chain),
+        "import_tiff_resized_scale_factors",
+    ],
+)
+def test_import_tiff_pipeline(folders, experiment_name):
+    # Can't use utility testing due to custom pipeline
+    config_filename = os.path.join(folders["config_folder"], experiment_name + ".json")
```

Comment:
My preference is f-strings when concatenating strings. But can also live with this

Reply:
changed to f-strings
they are a bit nicer indeed